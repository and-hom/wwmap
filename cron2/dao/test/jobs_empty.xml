<?xml version="1.0" ?>
<dataset>
    <job/>
</dataset>