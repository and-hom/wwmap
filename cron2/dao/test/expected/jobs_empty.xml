<?xml version="1.0" ?>
<dataset>
</dataset>